doc/howto.md

```diff
@@ -517,6 +517,54 @@ Content-Length: 3
 This same response can be used both for updates and queries for testing purposes (even though in the former the body won't
 be read).
 
+## IoT Agent in multi-thread mode
```

Comment:
New section should be included in the ToC in the top of the file.

Reply:
Fixed in 6433582